testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -61,6 +61,14 @@
             "changes": [
                 "Method added to public class"
             ]
+        },
```

Comment:
❓ This doesn't work, I did not have enough time to investigate why.

Reply:
I changed the `changes` as well from the generated snipped ... and that is obviously wrong 🤦